source/summary.rst

```diff
@@ -6,27 +6,40 @@
 Introduction
 ============
 
-We present and discuss new ways to prevent and detect active
+Autocrypt aims to achieve convenient end-to-end encryption of e-mail.
+The Level 1 Autocrypt specification offers users opt-in e-mail encryption,
+but only considers passive adversaries.
+Active network adversaries,
```

Comment:
I'd prefer the start to remain stating it's about preventing active attacks against autocrypt and only then explain autocrypt.